### DISCUSSION TITLE
Changelist TwinCAT 3.1.4024.29 to 4024.32

### DISCUSSION THREAD

**Discussion by TXAE** [Roald87/TwinCatChangelog#47] [General]
I'm just gonna leave this here...
[Changelist_4024.29_to_4024.32_1.pdf](https://github.com/Roald87/TwinCatChangelog/files/9169772/Changelist_4024.29_to_4024.32_1.pdf)

**Comment by PTKu** [comment]
@TXAE where did you get this? We've been asking for a document like this for years... 
Very valuable, thanks!

**Comment by Roald87** [comment]
Thanks for this list! I've added them to the changelog.
